conda_build/cli/main_index.py

```diff
@@ -66,6 +66,23 @@ def parse_args(args):
         action="store_true",
         dest="convert_if_not_present"
     )
+    p.add_argument(
+        "--gpg-sign",
+        help="Sign artifacts using GPG",
+        action="store_true",
+        dest="gpg_sign"
+    )
+    p.add_argument(
+        "--gpg-tool",
+        help="GPG program used to sign artifacts",
+        default="gpg",
+        dest="gpg_tool"
+    )
+    p.add_argument(
+        "--gpg-identity",
+        help="GPG identity used to sign artifacts",
+        dest="gpg_identity"
+    )
```

Comment:
The argument options are similar to the 'twine' utility, but with slightly modified names.